benchmarks/distributed/rpc/parameter_server/trainers/DdpRpcHelper.py

```diff
@@ -0,0 +1,48 @@
+from utils import sparse_tensor_to_rpc_format, sparse_rpc_format_to_tensor
+
+
+class DdpRpcHelper:
```

Comment:
curious, why does this needs to be a class instead of a helper function in utils?

Reply:
I think that the process_bucket_with_remote_server is specific to the class hierarchy of the current trainer implementations. It can be a helper method in utils. I will update the code.